README.md

```diff
@@ -1 +1,24 @@
-# Beyond-07-team-4
\ No newline at end of file
+# Beyond-07-team-4
+
+**Reqirements:**
+
+Adding a Vagrantfile for spinning up a development VM.
```

Comment:
what about git clone? of the project?
cd to the relevant directory, there is no vagrantfile in your readme later on



I would recommend the following steps:
Prerequisite:
VirtualBox 
vagrant

How to run project in vagrantbox
git clone project-url
cd project-dir

Reply:
Hope that in the updated README the changes you requested are done correctly.

Reply:
README file should contain specific and "to the point" about your project:
1- A phrase about this project
2- How a user not familiar with your project can clone and run 
3- what topics already implemented 
..

you can review some opensource projects, for example :  https://github.com/kubernetes/minikube

Reply:
+2 with @kasemAlem comments